addons/sale/views/sale_portal_templates.xml

```diff
@@ -143,7 +143,7 @@
                     </t>
                     <t t-set="entries">
                         <div class="d-flex flex-column gap-4">
-                            <div class="d-flex flex-column gap-1 order-md-last order-lg-first" id="sale_order_sidebar_button">
+                            <div class="d-flex flex-column gap-1" id="sale_order_sidebar_button">
```

Comment:
Since the button gets quite big on larger screens, I'd limit its width.
Something should also be done on mobile, imo :)

```diff 
-  <div class="d-flex flex-column gap-1" id="sale_order_sidebar_button">
+ <div class="d-flex flex-column gap-1 mw-lg-50" id="sale_order_sidebar_button">```